src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/MapFriendlyLocationController.kt

```diff
@@ -68,7 +90,7 @@ class MapFriendlyLocationController(private val service: MapFriendlyLocationServ
 
     service.mapFriendlyLocation(form.agencyId, form.locationName, LocationType.valueOf(form.locationType))
 
-    redirectAttributes.addFlashAttribute("flashAttrMappedLocationName", form.locationName)
+    redirectAttributes.addFlashAttribute("flashAttrMappedLocationName", form.locationName.toUpperCase())
```

Comment:
Small tweak as this are upper-cased when saved.  I guess ideally should come from the service but didn't seem worth it.